.github/workflows/ci.yaml

```diff
@@ -0,0 +1,31 @@
+name: scala-lang docs
+
+on:
+  pull_request:
+  schedule:
+    - cron: '00 0 * * *'
+
+jobs:
+  build-and-validate-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.5.8'
+          bundler-cache: true
+      - name: Build documents
+        run: |
+          bundle exec jekyll build 2> error.log
+          cat >&2 error.log
+          ( ! grep -qie Error -e Warn error.log )
+          find ./_site \( -type d \( -name events -o -name training \) -prune -false \) -o \( -type f ! -name error.log -empty \)
+          find ./_site \( -type d \( -name events -o -name training \) -prune -false \) -o \( -type f ! -name error.log -empty \) | ( ! read )
```

Comment:
Can we simplify this?
We could rely more on Github Actions logs rather than using custom text files that are not exported anywhere.

Reply:
It could be, but I would like to keep this PR as a straight rewrite with minimum changes